client/src/Pages/Profile/UserInfo/index.js

```diff
@@ -0,0 +1,124 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import axios from 'axios';
+import {
+  Typography,
+  Avatar,
+  Grid,
+  Paper,
+  CircularProgress,
+} from '@material-ui/core';
+import { Alert } from '@material-ui/lab';
+
+import Button from '../../../Components/Button';
+
+import useStyles from './style';
+
+function UserInfo({ getUserName }) {
+  const classes = useStyles();
+  const [user, setUser] = useState({});
+  const [loading, setLoading] = useState('');
+  const [errorMsg, setErrorMsg] = useState('');
+
+  const fetchingData = async (isCurrent) => {
```

Comment:
I think you should move this function definition inside the `useEffect` hook below and avoid passing `isCurrent` as an argument. At the moment it will always be `true` because it is copied when you call the function.